### ISSUE TITLE
Request: Delete the file on the specified day in the calendar (either a specific day or a week note).

### ISSUE THREAD

**Issue by deladriere** [bartkessels/daily-note-calendar#54]
I often use the right-click option on a date to delete a daily or weekly note in the LiamCain calendar. Would this feature be useful as well?

**Comment by bartkessels** [comment]
Hello, this is a great idea thanks for suggesting.

I've just checked out the LiamCain calendar, and I wondered if there were also other options you regularly use from the right-click menu?

Right now, I'm adding support for the delete action. But if you use other actions as well, just open another issue and it will be added in a future release.

As always, thanks for the feedback and if you have any more feature requests or bugs please let me know!

**Comment by deladriere** [comment]
Ok cool
Another question: Did you just remove the Shift-click on a date to display the note created on that date?

**Comment by bartkessels** [comment]
No, the shift + click should still only display the notes created on a date. The release that has been released was a bugfix so the dates are correctly aligned even when the daily note indicator is disabled. This was a small mistake from an earlier release.

But the shift + click doesn't work for you anymore?

**Comment by deladriere** [comment]
I confirm that shift-click doesn't work anymore. Also, I forgot where I learned about this feature; I can't see any reference on the options page. It could be good to edit the text in the settings.

**Comment by bartkessels** [comment]
Euhm, that is strange. Could you provide the settings you have set and on what OS you're using Obsidian? 'Cause I can't seem to reproduce the issue with the latest version. But that might also be because of another combination of settings that I'm using.

But we'll figure it out and create a bugfix release for this issue.

**Comment by deladriere** [comment]
I have the same issue on my two macOS 15.2 with a lot of plugins

Obsidian version: v1.8.0
Installer version: v1.4.13
Operating system: Darwin Kernel Version 24.2.0: Fri Dec  6 18:40:14 PST 2024; root:xnu-11215.61.5~2/RELEASE_ARM64_T8103 24.2.0

- Total startup time: 2,677ms
- Initialization: 404ms
- Vault (1,915 files): 284ms
- Workspace (17 tabs, 12 deferred): 741ms
- Core plugins: 99ms
- Community plugins (42 active): 1,150ms
  - Trash Explorer (v1.2.3): 273ms
  - Typewriter Scroll (v0.2.2): 180ms
  - Local REST API (v3.0.4): 77ms
  - Markmind (v3.1.0): 70ms
  - Chord Sheets (v1.5.2): 70ms
  - Day Planner (v0.27.0): 57ms
  - Kanban (v2.0.51): 52ms
  - Templater (v2.9.1): 44ms
  - Plugin Update Tracker (v1.6.0): 44ms
  - Dataview (v0.5.67): 37ms
  - QuickAdd (v1.11.5): 32ms
  - Tasks (v7.14.0): 24ms
  - Better Export PDF (v1.10.0): 22ms
  - Daily notes calendar (v1.11.1): 13ms
  - Natural Language Dates (v0.6.2): 12ms
  - Slash Commander (v0.2.16): 9ms
  - Archiver (v0.26.1): 9ms
  - Style Settings (v1.0.9): 8ms
  - Commander (v0.5.2): 8ms
  - Advanced URI (v1.44.3): 7ms
  - Editing Toolbar (v2.4.16): 6ms
  - Periodic Notes (v0.0.17): 6ms
  - Text Transporter (v1.0.12): 6ms
  - Easy Bake (v1.1.0): 6ms
  - Colored Tags (v5.0.1): 6ms
  - LanguageTool Integration (v0.3.7): 5ms
  - Handwritten Notes (v1.3.6): 5ms
  - Number Headings (v1.16.0): 4ms
  - Note Gallery (v0.0.60): 4ms
  - Outliner (v4.8.1): 4ms
  - Tag Wrangler (v0.6.1): 4ms
  - Note Definitions (v0.27.1): 3ms
  - File Info Panel (v1.3): 2ms
  - Recent Files (v1.7.3): 2ms
  - Remember cursor position (v1.0.9): 2ms
  - Ninja Cursor (v0.0.13): 2ms
  - Table of Contents (v0.2.0): 2ms
  - Metadata Extractor (v1.1.0): 2ms
  - Settings Search (v1.3.10): 1ms
  - Theme Toggler (v1.0.1): 1ms
  - Clear Unused Images (v1.1.1): 1ms
  - Focus Mode (v1.11.5): 1ms

**Comment by bartkessels** [comment]
Hello, do you perhaps use the setting to retrieve notes based on a property and have files with 'incorrect' frontmatter? Because issue #63 is registered with this issue. This will prevent the notes created on a specific date from being displayed.

**Comment by deladriere** [comment]
no property selected

**Comment by bartkessels** [comment]
On a lighter note, version 1.12.0 of the plugin has just been released. This allows you to delete periodic notes or notes that are displayed below the calendar through the plugin.

And as for your issue with shift + click goes, I'll keep searching for the bug and keep this issue opened until then.

**Comment by deladriere** [comment]
Delete works fine, thanks !

**Comment by bartkessels** [comment]
Hello, it's been a while.

I've updated the entire plugin which includes the way it handles the `shift` + click-action to only display the notes created on that date. Hopefully this will fix your issue, if not please let me know. In the upcoming release you'll be able to validate that it works (or not).

**Comment by deladriere** [comment]
Thanks : but now it displays the note created that days even if the slider  "Display notes created is selected date" is off
and without using shift click

**Comment by bartkessels** [comment]
Hello, as of version 2.4.1 of the plugin your latest issue with the "Display notes created on selected date" will be resolved.

As always thanks for your feedback and if you encounter any other bugs or have feature requests feel free to let me know!
